packages/express/src/express.ts

```diff
@@ -1,24 +1,19 @@
-import type { Logger, Plugin, TContextBase, TContextShape } from '@debugr/core';
+import type { CollectorPlugin, Logger, TContextBase, TContextShape } from '@debugr/core';
 import { LogLevel } from '@debugr/core';
-import type { HttpLogEntry } from '@debugr/http-common';
-import {
-  filterHeaders,
-  isCaptureEnabled,
-  normalizeContentLength,
-  normalizeContentType,
-} from '@debugr/http-common';
+import type { HttpRequestLogEntry, HttpResponseLogEntry } from '@debugr/http-common';
+import { normalizeContentLength } from '@debugr/http-common';
 import type { ErrorRequestHandler, Handler, Request, Response } from 'express';
 import type { NormalizedOptions, Options, ResponseInfo } from './types';
 import { normalizeOptions } from './utils';
 
 export class ExpressPlugin<
   TTaskContext extends TContextBase = TContextBase,
   TGlobalContext extends TContextShape = TContextShape,
-> implements Plugin<TTaskContext, TGlobalContext>
+> implements CollectorPlugin<TTaskContext, TGlobalContext>
 {
   public readonly id: string = 'express';
 
-  public readonly entryFormat = 'http' as const;
+  public readonly entryTypes: string[] = ['http.request', 'http.response'];
```

Comment:
tohle je strasne super. Tyyo multiple types

Reply:
jo, mě to nakonec přišlo čistší, takhle se třeba ty formattery nemusej zabejvat tím, jaký varianty struktury dat může ta "jejich" entry obsahovat, plus to potenciálně jde rozšiřovat o další věci ve stejným "namespace", pokud bychom nějaký našli (něco třeba jako `sql.connection`, kdybychom něco takovýho do budoucna vymysleli a chtěli)